test/pytorch_test_base.py

```diff
@@ -187,6 +187,15 @@
         'test_matrix_exp_analytic_xla',  # server side crash
         'test_muldiv_scalar_xla_bfloat16',  # FIXME
         'test_kthvalue_xla_.*',  # FIXME
+        'test_random_from_to_bool',  # doesn't raise
+        'test_random_from_to_xla',  # doesn't raise
+        'test_random_to_xla',  # doesn't raise
+        'test_random_default_xla_bfloat16',  # pt/xla uses larger default maxs for float types in random_
```

Comment:
Now I see what you meant with the test failure you reported 😑 
I wonder whether we should go along with this, or have `GetIntegerUpperLimitForType()` generate the pytorch max values for floating and use our MinMax API for all others.
@ailzhang Any thouths?

Reply:
Synced with @JackCaoG offline, we agreed that it's better to not skip these tests. @JackCaoG found out that he might be able to use `xla::bfloat16` to and copy the pytorch behavior in pytorch/xla. So let's see how that goes :D

Reply:
I updated the code to use `2^mantissa` as `to` when type is floating point types